### PR TITLE
✨ [Component] Add ISCN card

### PR DESCRIPTION
**Pull request by nwingt**
image

Todo:
- [x] Support responsive canvas
- [x] Include QR code
- [x] Change orientation

### CODE REVIEW

**Comment by AuroraHuang22** [comment]
🤩🤩🤩

**Review by williamchong — APPROVED** [review]
I still think a looping animation would be cool

**Review by AuroraHuang22 — APPROVED** [review]

**Review by williamchong — APPROVED** [review]
Would be nice if we can have lazy load later in search results

File: pages/embed/card/_iscnId.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  ) => Promise<{ records: ISCNRecordWithID[] }>
+
+  get record() {
+    return this.getISCNById(this.iscnId)
+  }
+
+  get isShowAdjustableProps() {
+    const { dev = '0' } = this.$route.query
+    return dev === '1'
+  }
```
Comment (williamchong) [inline pages/embed/card/_iscnId.vue:46]: Should this be behind `NODE_ENV !== 'production'` ?

Comment (nwingt) [reply pages/embed/card/_iscnId.vue:46]: This acts as an inspector, should come in handy even in production

**Pull request merged**